domain_whitelist.kv3

```diff
@@ -0,0 +1,14 @@
+// Bypass restricting web requests to this list of domains by adding the "-unrestrictedwebrequests" launch argument
+{
+	domains =
+	[
+		"steampowered.com",
+		"steamcommunity.com",
+		"valvesoftware.com",
```

Comment:
Since this is per-game (very happy it is), don't see the point of us supporting these domains

Reply:
Removed these domains